src/customfunctions/functions.js

```diff
@@ -0,0 +1,5 @@
+import * as MyFunctions from "./sample";
```

Comment:
this should import specific functions, and assign each function to CustomFunctionMappings.ID